CineElorrietaMVC/src/main/java/Controlador/ControladorPanelFacturas.java

```diff
@@ -1,70 +1,183 @@
 package Controlador;
 
 import Modelo.Modelo;
-import Modelo.Producto;
 import Modelo.ListaProductos;
 import Vista.PanelFacturas;
 import Vista.Vista;
 
+
 public class ControladorPanelFacturas {
```

Comment:
Una clase con 193 lineas es un disparate. además, la mitad de las cosas que estáis haciendo son bien transformaciones o verificaciones. Dividirlo en clases más pequeñas. Una clase no debería de exceder las 100 lineas. Y, una clase de 100 lineas ya es muy larga.

Reply:
Refactorizado

Reply:
169 lineas siguen siendo muchas